src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Easy.cs

```diff
@@ -0,0 +1,144 @@
+            internal const int CURLOPT_PROXYTYPE = CurlOptionLongBase + 101;
+            internal const int CURLOPT_HTTPAUTH = CurlOptionLongBase + 107;
+            internal const int CURLOPT_PROTOCOLS = CurlOptionLongBase + 181;
+            internal const int CURLOPT_REDIR_PROTOCOLS = CurlOptionLongBase + 182;
+
+            internal const int CURLOPT_WRITEDATA = CurlOptionObjectPointBase + 1;
+            internal const int CURLOPT_URL = CurlOptionObjectPointBase + 2;
+            internal const int CURLOPT_PROXY = CurlOptionObjectPointBase + 4;
+            internal const int CURLOPT_PROXYUSERPWD = CurlOptionObjectPointBase + 6;
+            internal const int CURLOPT_READDATA = CurlOptionObjectPointBase + 9;
+            internal const int CURLOPT_COOKIE = CurlOptionObjectPointBase + 22;
+            internal const int CURLOPT_HTTPHEADER = CurlOptionObjectPointBase + 23;
+            internal const int CURLOPT_HEADERDATA = CurlOptionObjectPointBase + 29;
+            internal const int CURLOPT_CUSTOMREQUEST = CurlOptionObjectPointBase + 36;
+            internal const int CURLOPT_ACCEPT_ENCODING = CurlOptionObjectPointBase + 102;
+            internal const int CURLOPT_PRIVATE = CurlOptionObjectPointBase + 103;
+            internal const int CURLOPT_COPYPOSTFIELDS = CurlOptionObjectPointBase + 165;
+            internal const int CURLOPT_SEEKDATA = CurlOptionObjectPointBase + 168;
+            internal const int CURLOPT_USERNAME = CurlOptionObjectPointBase + 173;
+            internal const int CURLOPT_PASSWORD = CurlOptionObjectPointBase + 174;
+
+            internal const int CURLOPT_WRITEFUNCTION = CurlOptionFunctionPointBase + 11;
+            internal const int CURLOPT_READFUNCTION = CurlOptionFunctionPointBase + 12;
+            internal const int CURLOPT_HEADERFUNCTION = CurlOptionFunctionPointBase + 79;
+            internal const int CURLOPT_SSL_CTX_FUNCTION = CurlOptionFunctionPointBase + 108;
+            internal const int CURLOPT_SEEKFUNCTION = CurlOptionFunctionPointBase + 167;
+        }
+
+        // Class for constants defined for the enum CURLINFO in curl.h
+        internal static partial class CURLINFO
+        {
+            // Curl info are of the format <type base> + <n>
+            private const int CurlInfoStringBase = 0x100000;
+            private const int CurlInfoLongBase = 0x200000;
+
+            internal const int CURLINFO_PRIVATE = CurlInfoStringBase + 21;
+            internal const int CURLINFO_HTTPAUTH_AVAIL = CurlInfoLongBase + 23;
+        }
+
+        // AUTH related constants
+        internal static partial class CURLAUTH
+        {
+            internal const long None = 0;
+            internal const long Basic = 1 << 0;
+            internal const long Digest = 1 << 1;
+            internal const long Negotiate = 1 << 2;
+        }
+
+        // Class for constants defined for the enum curl_proxytype in curl.h
+        internal static partial class curl_proxytype
```

Comment:
These are the same casings as used in curl.h:
```C++
typedef enum {
  CURLPROXY_HTTP = 0,   /* added in 7.10, new in 7.19.4 default is to use
....
} curl_proxytype;  /* this enum was added in 7.10 */

  /* Fill in new entries below here! */
typedef enum {
  CURLINFO_NONE, /* first, never use this */
....
  CURLINFO_LASTONE          = 43
} CURLINFO;

typedef enum {
  /* This is the FILE * or void * the regular output should be written to. */
  CINIT(FILE, OBJECTPOINT, 1),

  /* The full URL to get/put */
  CINIT(URL,  OBJECTPOINT, 2),
...
  CURLOPT_LASTENTRY /* the last unused */
} CURLoption;
```

Should we be deviating from that?